packages/driver/test/cypress/integration/commands/xhr_spec.js

```diff
@@ -2007,6 +2007,9 @@ describe('src/cy/commands/xhr', () => {
       })
 
       it('sets err on log when caused by the XHR response', function (done) {
+        // NOTE: sometimes the .then command will timeout before the request finishes
+        // so bump the timeout here. This does not increase test duration
+        Cypress.config('defaultCommandTimeout', 1000)
```

Comment:
this will set it for the remaining specs in this file, which feels wrong, shouldn't it be reset somewhere? or use this per-test config thing i keep hearing about

Reply:
no, it will not set config on the remaining tests, since we reset Cypress.config in a beforeEach in our support/defaults file